Makefile

```diff
@@ -14,6 +14,8 @@
 
 .PHONY: all rbdplugin cephfsplugin
 
+DOCKERCMD:=docker
```

Comment:
```suggestion
CONTAINER_RUNTIME_BINARY:=docker
```

Reply:
I can make that change but I chose 'DOCKERCMD' because it matches the variable name in a similar change I made in rook. Let me know if you feel strongly about it though.

Reply:
Not really but if `DOCKERCMD` is `podman` then that does not mean much at the end ^^. That's why I proposed `CONTAINER_RUNTIME_BINARY` to make it as generic as possible.

Reply:
Sure. :-) But docker is the defacto standard and podman is compatible with docker cli.